tests/helpers/test_service.py

```diff
@@ -185,22 +198,21 @@ def test_fail_silently_if_no_service(self, mock_log):
     hass.states.async_set("light.Kitchen", STATE_OFF)
 
     await hass.components.group.Group.async_create_group(
-        hass, "test", ["light.Ceiling", "light.Kitchen"]
-    )
+        hass, "test", ["light.Ceiling", "light.Kitchen"])
 
     call = ha.ServiceCall("light", "turn_on", {ATTR_ENTITY_ID: "light.Bowl"})
 
     assert {"light.bowl"} == await service.async_extract_entity_ids(hass, call)
 
     call = ha.ServiceCall("light", "turn_on", {ATTR_ENTITY_ID: "group.test"})
 
-    assert {"light.ceiling", "light.kitchen"} == await service.async_extract_entity_ids(
-        hass, call
-    )
+    assert {"light.ceiling", "light.kitchen"
+            } == await service.async_extract_entity_ids(hass, call)
 
-    assert {"group.test"} == await service.async_extract_entity_ids(
-        hass, call, expand_group=False
-    )
+    assert {"group.test"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4603109)